classes/task/ReviewReminder.inc.php

```diff
@@ -56,70 +62,41 @@ public function sendReminder($reviewAssignment, $submission, $context, $reminder
             return false;
         }
 
-        $emailKey = $reminderType;
+        $mailable = $this->getMailable($reminderType, $reviewAssignment, $submission, $context);
```

Comment:
I'd suggest the same thing about splitting things out from the helper function so that you can better type hint what mailables you are working with. Especially since `$reminderType` can be anything passed in by code that calls this function, you may want to guard against a misuse by checking the type against valid types or something.